wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,24 +1,6 @@
 {
   "object": {
     "pins": [
-      {
```

Comment:
should this be removed ?

Reply:
this is a mistake, thanks! 3b0cd5755d61bb3437b49779492ab848d59e1148